### PR TITLE
Update references to ref fields spec

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Review by jaredpar — APPROVED** [review]

File: src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs [hunk 1]
```diff
@@ -1542,11 +1542,10 @@ private BoundExpression InferTypeForDiscardAssignment(BoundDiscardExpression op1
                 {
                     if (isRef)
                     {
-                        // https://github.com/dotnet/csharplang/blob/main/proposals/low-level-struct-improvements.md#rules-ref-reassignment
-                        // For a ref reassignment in the form ...
-                        // 1. x.e1 = ref e2: where x is safe-to-escape at least return only then e2 must have ref-safe-to-escape at least as large as x
-                        // 2. e1 = ref e2: where e1 is a ref local or ref parameter then e2 must have a safe-to-escape equal to safe-to-escape for e1 and
-                        //    e2 must have ref-safe-to-escape at least as large as ref-safe-to-escape of the ref-safe-to-escape of e1
+                        // https://github.com/dotnet/csharplang/blob/main/proposals/csharp-11.0/low-level-struct-improvements.md#rules-ref-reassignment
+                        // For a ref reassignment in the form `e1 = ref e2` both of the following must be true:
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs:1546]: Thanks! :-)

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 1)

File: src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs (hunk 1 above)
Comment (Therzok) [reply src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs:1546]: I think that permalinking the commit (y hotkey) will fix the issue of the file being moved.

You can then navigate to the file history from that permalink.

Comment (cston) [reply src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs:1546]: Thanks @Therzok for the suggestion. I'll keep the links as is though, so the links refer to the latest text.

**Pull request merged**